tests/integration/examples/test_driver_introduction_example.py

```diff
@@ -27,6 +27,7 @@
 # tag::driver-introduction-example-import[]
 from neo4j import GraphDatabase
 import logging
+import sys
 from neo4j.exceptions import ServiceUnavailable
```

Comment:
```suggestion
import logging
import sys

from neo4j import GraphDatabase
from neo4j.exceptions import ServiceUnavailable
```

First import sys modules, then 3rd party modules, then local modules. Each block in alphabetical order and separated by a blank line.